server/src/models/Board.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const BoardSchema = new Schema({
+    user: {
+        type: Schema.Types.ObjectId,
+        ref: 'users'
+    },
+    title: {
+        type: String,
+        required: true
+    },
+    posts: [
+        {
+            type: Schema.Types.ObjectId,
+            ref:'post'
+        }
+    ],
+    date: {
+        type: Date,
+        default: Date.now()
+    }
+})
+
```

Comment:
I would suggest using virtual populate for the one to many relationship with board to posts. May make querying easier.

BoardSchema.virtual('posts', {
  ref: 'PostSchema',
  localField: '_id',
  foreignField: 'board'
});